lib/MusicBrainz/Server/Entity/Editor.pm

```diff
@@ -102,13 +102,18 @@ sub is_editing_disabled {
 }
 
 sub is_editing_enabled {
-    (shift->privileges & $EDITING_DISABLED_FLAG) == 0;
+    my $mask = $EDITING_DISABLED_FLAG ^ $SPAMMER_FLAG;
```

Comment:
It's kinda odd how `is_editing_disabled` is now not always equal to the negation of `is_editing_enabled`.  I'd expect that to be true.

Is there any downside to applying the same logic to `is_editing_disabled` too?  Or even `is_adding_notes_disabled` for that matter?

Alternatively, would it make sense to just set `$EDITING_DISABLED_FLAG` and `$ADDING_NOTES_DISABLED_FLAG` automatically whenever the spammer flag is added?

Reply:
I implemented the latter suggestion as per our IRC chat.